src/KingSystem/Sound/sndMusicInfo.cpp

```diff
@@ -0,0 +1,52 @@
+#include "KingSystem/Sound/sndMusicInfo.h"
+
+namespace ksys::snd {
+
+MusicParam::OtherSub::OtherSub()
+    : mBPM(0x6E, "Bpm", "BPM", "Min=60,Max=180", this),
+      mPart0FirstWaitCount(0, "Part0FirstWaitCount", "パート0 初回予約の待ち拍", "Min=0,Max=100",
+                           this),
+      mPart1FirstWaitCount(1, "Part1FirstWaitCount", "パート1 初回予約の待ち拍", "Min=0,Max=100",
+                           this),
+      mPart2FirstWaitCount(15, "Part2FirstWaitCount", "パート2 初回予約の待ち拍", "Min=0,Max=100",
+                           this),
+      mPart3FirstWaitCount(30, "Part3FirstWaitCount", "パート3 初回予約の待ち拍", "Min=0,Max=100",
+                           this) {}
+
+MusicParam::MusicParam()
+    : mEnableWaitSec(20, "EnableWaitSec", "昼夜切り替えや条件外れた後の開始待ち秒",
+                     "Min=0.f, Max=40.f", &mData),
+      mDistanceOfCheckingActiveEnemy('F', "DistanceOfCheckingActiveEnemy",
```

Comment:
```suggestion
      mDistanceOfCheckingActiveEnemy(70, "DistanceOfCheckingActiveEnemy",
```
(F doesn't really make sense here)